docs/content/docs/flink/flink-getting-started.md

```diff
@@ -481,7 +481,7 @@ stream.print();
 env.execute("Test Iceberg Batch Read");
 ```
 
-There are other options that we could set by Java API, please see the [FlinkSource#Builder](../javadoc/{{% icebergVersion %}}/org/apache/iceberg/flink/source/FlinkSource.html).
```

Comment:
Can we deploy Javadocs in the same locations so that we don't break the existing URIs?

Reply:
Ah, you mean `javadoc` as opposed to `javadocs`? Thanks for catching that, not sure how I missed that!

Reply:
Fixed! I also added a redirect for `iceberg.apache.org/javadoc` -> `iceberg.apache.org/javadoc/latest`

Reply:
We also need to make sure that none of the existing Javadoc links break.